### DISCUSSION TITLE
Spawn vs invoking

### DISCUSSION THREAD

**Discussion by dsathyakumar** [statelyai/xstate#2093] [Q&A] [answered]
Apologies if my question is dumb. 🙈 
I'm trying to understand the different usecases where `spawn` would be preferred over `invoking`, or broadly, what is the difference between the two ? 🤔

**Comment by dsathyakumar** [comment, marked as answer]
Ah I found this. 
https://spectrum.chat/statecharts/general/how-does-invoking-services-compare-to-spawning-actors-in-xstate~3ee67a83-26b2-4a24-a0c6-fbdce46f574a
Thanks 👍

**Comment by redbar0n** [comment]
See also https://stackoverflow.com/a/66720053/380607
